base62.py

```diff
@@ -40,6 +72,25 @@ def decode(b):
 
     return v
 
+
+def decodebytes(s):
+    """Decodes a bytestring of base62 data into a bytes object.
+
+    :param s: A bytestring to be decoded in base62
+    :rtype: bytearray
+    """
+    _check_bytes_type(s)
+
+    decoded = decode(s.decode(DEFAULT_ENCODING))
+    buf = []
+    while decoded > 0:
+        buf.append(decoded & 0xff)
+        decoded //= 256
+    buf.reverse()
+
+    return bytearray(buf)
```

Comment:
I think you should wrap the `bytearray` in `bytes()` before returning. This will result in the desired behaviors for Python 2 & 3.